### PR TITLE
Updated callback to route messages to AWS Connect queue for routing into NYS call center

### PR DESCRIPTION
**Pull request by dennisgove**

### CODE REVIEW

**Comment by dennisgove** [comment]
A few changes have come out of a discussion with Jesse at AWS.

In our initial conversations they weren't aware that we had a lambda already. What they suggested at first can be simplified a lot because we're running inside a lambda. Instead of hitting a HTTP endpoint we can actually add the callback to New York State's callback connect process directly. We don't need to go through this extra authentication and HTTP hop.

Jesse is sending me sample code for adding a call directly to the Connect API. We looked at it together and it's very straight forward.

File: utils.js [hunk 1]
```diff
@@ -30,21 +30,22 @@ async function getAssetsBucket() {
...
-      sig: process.env.CCT_SIG,
-      queueUrl: process.env.CALLBACK_QUEUE_URL
+      awsConnectInstanceId: process.env.AWS_CONNECT_INSTANCE_ID,
+      awsConnectContactFlowId: process.env.AWS_CONNECT_CONTACT_FLOW_ID,
+      awsConnectQueueId: process.env.AWS_CONNECT_QUEUE_ID,
+      awsConnectApiEntryPhoneNumber: process.env.AWS_CONNECT_API_ENTRY_PHONE_NUMBER,
+      callbackQueueUrl: process.env.CALLBACK_QUEUE_URL,
+      awsConnectCrossAccountDestinationAccountId: process.env.AWS_CONNECT_CROSS_ACCOUNT_DESTINATION_ACCOUNT_ID,
+      awsConnectCrossAccountRoleSessionName: process.env.AWS_CONNECT_CROSS_ACCOUNT_ROLE_SESSIOON_NAME,
+      awsConnectCrossAccountExternalId: process.env.AWS_CONNECT_CROSS_ACCOUNT_EXTERNAL_ID,
```
Comment (segfault) [inline utils.js:48]: Looks like a typo here `process.env.AWS_CONNECT_CROSS_ACCOUNT_ROLE_SESSIOON_NAME`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,63 @@
...
+#### Secret Settings
+
+| .env | AWS Secret Key | Field | Meaning
+| :--- | :--- | :--- | :--- |
+| `AWS_CONNECT_INSTANCE_ID` | `awsPostCallback` | `awsConnectInstanceId` | ID of New York State's AWS Connect Instance |
+| `AWS_CONNECT_CONTACT_FLOW_ID` | `awsPostCallback` | `awsConnectContactFlowId` | ID of the AWS Connect Contact Flow |
+| `AWS_CONNECT_QUEUE_ID` | `awsPostCallback` | `awsConnectQueueId` | ID of the AWS Connect Queue callback requests are added to |
+| `AWS_CONNECT_API_ENTRY_PHONE_NUMBER` | `awsPostCallback` | `awsConnectApiEntryPhoneNumber` | Fake call center phone number used to post callback requests to the AWS Connect API |
+| `AWS_CONNECT_CROSS_ACCOUNT_DESTINATION_ACCOUNT_ID` | `awsPostCallback` | `awsConnectCrossAccountDestinationAccountId` | NYS Call Center AWS Account Id |
+| `AWS_CONNECT_CROSS_ACCOUNT_ROLE_SESSIOON_NAME` | `awsPostCallback` | `awsConnectCrossAccountRoleSessionName` | ?? |
```
Comment (segfault) [inline README.md]: s/SESSIOON/SESSION/

**Review by segfault — CHANGES_REQUESTED** [review]
looks good. small typo to fix up

**Pull request merged**